docs/student/student-schema.yaml

```diff
@@ -0,0 +1,59 @@
+definitions:
+  students:
+    type: array
+    items:
+      type: object
+      properties:
+        _id:
+          type: string
+        role:
+          type: string
+          enum:
```

Comment:
mby we should stick to the same pattern for enums? and it should have only student tutor methinks
enum: [student, tutor]

Reply:
fixed